Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN ln -s /mnt/bes/pcm $HOME/ScenarioData
 
 COPY powersimdata/utility/templates /mnt/bes/pcm/
 
-WORKDIR /app
+WORKDIR /PowerSimData
```

Comment:
This is unrelated, but wanted to keep it. Has no effect except in docker images based on this one.